### ISSUE TITLE
use friendica profile 'about' field as diaspora 'bio' field

### ISSUE THREAD

**Issue by haheute** [friendica/friendica#13640]
### Is the feature request related to a problem? Please describe.

When I add a small description of me or some text in the profile settings `about` field, it seems that this text is not visible on diaspora.

### Describe the feature you'd like

Maybe the friendica `about` text could be sent to diaspora and be used there in the `bio` field of the user profile.

### Describe alternatives you've considered

none. or let it be as it is

### Additional context

none

**Comment by haheute** [comment]
Thanks. It seems in my diaspora account, my friendica profile still doesn't show the description. I don't know how it is updated. If that is friendica or if diaspora needs to do something..

**Comment by annando** [comment]
It is only updated upon change.

**Comment by haheute** [comment]
Hmm.. I think I had removed the second sentence from my description field some weeks ago (to trigger some update).
On [diaspora.schoenf.de](https://diaspora.schoenf.de/people/172731f5-1863-720c-6f54-36c608422786) I still don't see any description or about text on my Friendica (boerdica.de) account. I think I have also unfriended my boerdica.de account on diaspora and added it again, to see if that would help.. (it did not).

I have now again changed my description text  (added two dots ( `..` ). And will observe the case ;-)
https://boerdica.de/profile/noidea

On libranet.de I can already see the change. On diaspora not yet.
